clients/src/main/java/org/apache/kafka/clients/consumer/internals/FetchConfig.java

```diff
@@ -54,11 +49,8 @@
  *
  * Note: the {@link Deserializer deserializers} used for the key and value are not closed by this class. They should be
  * closed by the creator of the {@link FetchConfig}.
- *
```

Comment:
There is still a mention of `deserializers` above.

Reply:
Will exorcise them.